src/components/pages/blog-index.svelte

```diff
@@ -44,6 +44,10 @@
       ''
     );
 
+    if (tags.length === 0) {
+      return $page.data.page?.story?.content?.seo_title;
+    }
+
```

Comment:
The only nit picky comment I have is that we could do this check before creating the titlePrefix variable, since the titlePrefix will only be used if there are tags present.
I'm approving it either way since it's not a deal breaker